### PR TITLE
Use http for sandbox to allow LTI endpoints to work

### PR DESCRIPTION
**Pull request by kluo**
@stvstnfrd 
Edx config wasn't writing this setting to envs file, [PR'ed](https://github.com/edx/configuration/pull/5416) and cherry-picked here

### CODE REVIEW

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -1618,6 +1619,7 @@ lms_env_config:
   OAUTH_DELETE_EXPIRED: "{{ EDXAPP_OAUTH_DELETE_EXPIRED }}"
   PAID_COURSE_REGISTRATION_CURRENCY: "{{ EDXAPP_PAID_COURSE_REGISTRATION_CURRENCY }}"
   GIT_REPO_DIR: "{{ EDXAPP_GIT_REPO_DIR }}"
+  HTTPS: "{{ EDXAPP_LMS_HTTPS }}"
```
Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: Can you move this up into the the section marked
`# Stanford Forked Values: Start`?

File: playbooks/roles/edxapp/defaults/main.yml [hunk 2]
```diff
@@ -486,6 +486,7 @@ EDXAPP_FEATURES: &edxapp_features
 # as the lms (it's sent in the request)
 EDXAPP_SITE_NAME: 'localhost'
 EDXAPP_LMS_SITE_NAME: "{{ EDXAPP_SITE_NAME }}"
+EDXAPP_LMS_HTTPS: 'on'
```
Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: Same here, can we move it to the Stanford block of vars?

File: stanford/playground/group_vars/all [hunk 3]
```diff
@@ -23,6 +23,7 @@ EDXAPP_FEATURES_ENABLE_COURSEWARE_SEARCH: true
 EDXAPP_FEATURES_ENABLE_DASHBOARD_SEARCH: true
 EDXAPP_LMS_BASE: "{{ EDXAPP_SITE_NAME }}"
 EDXAPP_LMS_ENABLE_BASIC_AUTH: True
+EDXAPP_LMS_HTTPS: 'off'
```
Comment (stvstnfrd) [inline stanford/playground/group_vars/all]: This will _not_ break in devstack because it's explicitly overridden in `devstack.py` 👍

**Comment by kluo** [comment]
@stvstnfrd I was thinking this change could also go upstream so that's why I intentionally placed it outside the Stanford block, or is it always preferred to add it to the forked values block and then deal with an upstream merge redundancy later?

**Comment by stvstnfrd** [comment]
@kluo I guess it comes down to a matter of  _when_ will we PR upstream.

If I'm not PRing and cherry-picking back down immediately (literally, PRing upstream prior to origin), my preference is usually to just treat it as a fork because all too often, "we should eventually PR this"-type changes never make it upstream.

Side note: please feel encouraged to err on the side of including _too_ _much_ information on PRs.
They need not just be "what" changed, but also:
- _why_ do we need this (eg: "edx wasn't parsing this config value from JSON")
- why did you implement it _like_ _this_ and not _like_ _that_
- what are our future intentions for this? (eg: "we should PR this upstream")
- etc.

It not only saves some back-and-forth, but also sheds light into your thought process, which is helpful both now on this PR, as well as in retrospect upon `git blame`, etc.

**Pull request merged**